pyaerocom/data/paths.ini

```diff
@@ -100,6 +100,7 @@ EEA_V2 = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/EEA_AQeRep.v2/renamed/
 
 AIR_NOW = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/MACC_INSITU_AirNow
 MARCO_POLO = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/CHINA_MP_NRT
+MEP = ${BASEDIR}/aerocom/aerocom1/AEROCOM_OBSDATA/MEP/download/
```

Comment:
Does this need to be here?
It would be better if this could be defined on the plugin.